src/report_workflow/test_report_runner.py

```diff
@@ -48,6 +52,14 @@ def __init__(self, args: ReportArgs, test_manifest: TestManifest) -> None:
 
     def update_data(self) -> dict:
         self.test_run_data["name"] = self.product_name
+        if float(self.schema_version) > 1.0:
+            self.bundle_manifest = BundleManifest.from_urlpath(self.dist_manifest)
+            self.test_run_data["version"] = self.bundle_manifest.build.version
+            self.test_run_data["platform"] = self.bundle_manifest.build.platform
+            self.test_run_data["architecture"] = self.bundle_manifest.build.architecture
+            self.test_run_data["distribution"] = self.bundle_manifest.build.distribution
+            self.test_run_data["id"] = self.bundle_manifest.build.id
+            self.test_run_data["rc"] = self.release_candidate
```

Comment:
Same question here. The RC seems to be the same as `id` right? I think we don't need the argument for RC.

Reply:
Same as above.